### PR TITLE
Show disabling secondary index

### PR DESCRIPTION
**Pull request by WilliamBZA**
Fixes https://github.com/Particular/docs.particular.net/issues/5176

Shows an example of disabling the secondary index to make it clearer for users how to access the `Compatibility()` function.

### CODE REVIEW

File: samples/rabbitmq/simple/Rabbit_7/Sender/MyCommandHandler.cs [hunk 1]
```diff
@@ -0,0 +1,28 @@
+﻿using System;
+using System.Threading;
+using System.Threading.Tasks;
+using NServiceBus;
+using NServiceBus.Logging;
+
+public class MyCommandHandler : IHandleMessages<MyCommand>
```
Comment (mikeminutillo) [inline samples/rabbitmq/simple/Rabbit_7/Sender/MyCommandHandler.cs:7]: Is this related to the same feedback? I'm not sure why this handler is here.

**Review by danielmarbach — APPROVED** [review]
